modules/ref/pages/travel-app-data-model.adoc

```diff
@@ -3,6 +3,6 @@
 :page-topic-type: reference
 
 [abstract]
-The Travel Sample Application is currently available in 6 of our SDKs.
+Coming soon
```

Comment:
Tis done, no?

Reply:
yes! Maria is working on the docs page, I should probably have left it unchanged to avoid confusion :)